### ISSUE TITLE
Completion with StarCoder2-7B in VSCode shows unexpected tags

### ISSUE THREAD

**Issue by CleyFaye** [TabbyML/tabby#2380]
**Describe the bug**
When using the TabbyML VSCode extension, completion suggestion shows a bunch of tags/stuff after the actual completion.
![image](https://github.com/TabbyML/tabby/assets/1065984/1c7b7194-bd6f-47ee-b4b5-b5b71d33696a)

This is for completing TypeScript code using StarCoder2-7B

**Information about your version**
Tabby version: 0.11.1 running on Windows (tabby_x86_64-windows-msvc-cuda122)
VSCode extension 1.6.3

**Information about your GPU**
Output of nvidia-smi:
```raw
+-----------------------------------------------------------------------------------------+
| NVIDIA-SMI 552.12                 Driver Version: 552.12         CUDA Version: 12.4     |
|-----------------------------------------+------------------------+----------------------+
| GPU  Name                     TCC/WDDM  | Bus-Id          Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
|                                         |                        |               MIG M. |
|=========================================+========================+======================|
|   0  NVIDIA GeForce RTX 3080 Ti   WDDM  |   00000000:2D:00.0  On |                  N/A |
|  0%   51C    P8             35W /  370W |    1234MiB /  12288MiB |      9%      Default |
|                                         |                        |                  N/A |
+-----------------------------------------+------------------------+----------------------+
```

**Additional context**
Tabby runs on a remote windows system, while VSCode is on a Linux system. This should not be an issue, but it's worth mentionning.

The command used to run tabbyml is `tabby_x86_64-windows-msvc-cuda122.exe serve --model StarCoder2-7B --device cuda`

**Comment by wsxiaoys** [comment]
Seems to be a dup of https://github.com/TabbyML/tabby/issues/2277 (fixed in 0.12)

**Comment by CleyFaye** [comment]
Ah, my bad; I completely missed that release. Sorry.
